### PR TITLE
Changed merge policy

### PR DESCRIPTION
**Pull request by piosz**
Follow up of our discussion with Tim.

cc @thockin @brendandburns @rsokolowski @fgrzadkowski @wojtek-t @jszczepkowski @gmarek

### CODE REVIEW

**Comment by zmerlynn** [comment]
Just to be clear, are we saying that merges are now a 24/5 free-for all with no core hours, as long as they've simmered for 2 hours? Or that people can only merge when it's daylight relative to them? (Which is super confusing because it's 6am here and core hours in Poland right now.)

**Comment by zmerlynn** [comment]
(The "business day" language threw me off. I'm assuming that just means M-F and isn't related to hours.)

**Comment by piosz** [comment]
@zmerlynn In my opinion the first option is better as long as merger will be available for some time to fix possible problem. Let's see what other folks think.

"business day" means M-F excepting holidays. If you think there is a better word, I won't argue.

**Comment by zmerlynn** [comment]
Nah, I'm happy with 24/5. Y'all have also been pretty carefully watching
Jenkins, which makes me happy.

Business Day is fine. "Monday-Friday" would be precise and simple, too.

File: docs/devel/collab.md [hunk 1]
```diff
@@ -6,13 +6,9 @@ Kubernetes is open source, but many of the people working on it do so as their d
...
-
 ## Code reviews
 
-All changes must be code reviewed.  For non-maintainers this is obvious, since you can't commit anyway.  But even for maintainers, we want all changes to get at least one review, preferably from someone who knows the areas the change touches.  For non-trivial changes we may want two reviewers.  The primary reviewer will make this decision and nominate a second reviewer, if needed.  Except for trivial changes, PRs should sit for at least 2 hours to allow for wider review.
+All changes must be code reviewed.  For non-maintainers this is obvious, since you can't commit anyway.  But even for maintainers, we want all changes to get at least one review, preferably (for non-trivial changes obligately) from someone who knows the areas the change touches.  For non-trivial changes we may want two reviewers.  The primary reviewer will make this decision and nominate a second reviewer, if needed.  Except for trivial changes, PRs should sit for at least 2 hours to allow for wider review.
```
Comment (thockin) [inline docs/devel/collab.md]: Propose to strike the "at least 2 hours" language and replace with 

```
Except for trivial changes, PRs should not be committed until relevant parties (e.g. owners of the subsystem affected by the PR) have had a reasonable chance to review them.
```

File: docs/devel/collab.md [hunk 2]
```diff
@@ -28,7 +24,7 @@ Maintainers can assign reviews to other maintainers, when appropriate. The assig
 
 ## Merge hours
 
-Maintainers will do merges between the hours of 7:00 am Monday and 7:00 pm (19:00h) Friday.  PRs that arrive over the weekend or on holidays will only be merged if there is a very good reason for it and if the code review requirements have been met.
+Maintainers will do merges during business days (according to US calendar). PRs that arrive over the weekend or on holidays will only be merged if there is a very good reason for it and if the code review requirements have been met.
```
Comment (thockin) [inline docs/devel/collab.md]: This reads as the same as what you changed it from, but less precisely.  How about:

```
Maintainers will do merges of appropriately reviewed-and-approved changes during their local "business hours" (typically 7:00 am Monday to 5:00 pm (17:00h) Friday).  PRs that arrive over the weekend or on holidays will only be merged if there is a very good reason for it and if the code review requirements have been met.  Concretely this means that nobody should merge changes immediately before going to bed for the night.
```

**Comment by thockin** [comment]
Let's de-emphasize exact times and instead focus on intent - relevant parties must sign off on important PRs.  The less complex the PR, the less the rules matter.  Nobody should be merging on their own nights and weekends.  Nobody should be merging without giving people a reasonable shot at review.

File: docs/devel/collab.md (hunk 1 above)
Comment (piosz) [inline docs/devel/collab.md]: Done

File: docs/devel/collab.md (hunk 2 above)
Comment (piosz) [inline docs/devel/collab.md]: Done

**Comment by brendandburns** [comment]
Yeah, I'd summarize the intent "Relevant parties should have the opportunity to look at a PR in their local business hours" or some such.  The goal is that no one feel they need to check the open PR list at all hours.

Can you add a sentence at the start to indicate something along these lines? 

Thanks!
--brendan

**Comment by brendandburns** [comment]
LGTM.  Will merge during business hours.  (or feel free to do the same ;)

**Pull request merged**